makefile

```diff
@@ -171,8 +172,10 @@ vendor-clean:
 	rm -rf vendor/github.com/mongodb/grip/vendor/github.com/google/uuid/
 	rm -rf vendor/github.com/mongodb/grip/vendor/github.com/pkg/
 	rm -rf vendor/github.com/mongodb/grip/vendor/github.com/stretchr/testify/
-	rm -rf vendor/github.com/mongodb/amboy/vendor/github.com/google/uuid
 	rm -rf vendor/github.com/mongodb/grip/vendor/github.com/google/uuid
+	# riscv64 isn't recognized as a valid GOARCH until go1.16, so we can likely stop removing this file once CI tests
```

Comment:
Can you link a ticket? And maybe make it a TODO?

Reply:
[EVG-14482](https://jira.mongodb.org/browse/EVG-14482)